mailpoet/lib/Validator/ValidationException.php

```diff
@@ -10,8 +10,10 @@ class ValidationException extends UnexpectedValueException {
   protected $wpError;
 
   public static function createFromWpError(WP_Error $wpError): self {
-    $exception = self::create()
-      ->withMessage($wpError->get_error_message());
```

Comment:
Should we maybe keep both the message and errors?

Reply:
You mean something like `self::create()->withMessage()->withWpError($wpError)`?

Currently I am not sure if we need it and we can always add it later on I guess.